### ISSUE TITLE
Typography doesn't support $rhythm-unit

### ISSUE THREAD

**Issue by jakobrosenberg** [Igosuki/compass-mixins#33]
Any chance you could update the typography module to support $rhythm-unit?
http://compass-style.org/reference/compass/typography/vertical_rhythm/

Thank you for this project. It's much appreciated.

**Comment by jasunde** [comment]
I agree. Looks like the '$rhythm-unit' variable has been replaced by '$font-unit'.

**Comment by tonijn** [comment]
it seems line 126 is giving problems because it can't convert em units to px
